app.rb

```diff
@@ -0,0 +1,49 @@
+require 'sinatra'
+require 'sinatra/reloader'
+require 'active_record'
+
+# Load the file to connect to the DB
+require_relative 'db/connection'
+
+# Load models
+require_relative 'models/pokemon'
+
+get '/' do
+  @pokemonsvariable=Pokemon.all
```

Comment:
`@pokemons` would be a more semantically accurate name. You don't need to include the word "variable" because we already know this is a variable!